### ISSUE TITLE
UI Refactoring of Stopwatch 

### ISSUE THREAD

**Issue by kaxcode** [kaxcode/cloudwatch#121]

**Comment by meingorn** [comment]
keeping this open for now - buttons will need refactoring once everything is merged

**Comment by kaxcode** [comment]
Buttons are in both Stopwatch and Timer so it should be solved in a separate issue.
